rpm/0089-sailfishos-gecko-Add-a-video-decoder-based-on-gecko-.patch

```diff
@@ -287,8 +306,8 @@ index 000000000000..8dc4ca3f32b0
 +                 : ConversionRequired::kNeedNone;
 +}
 +
-+void GeckoCameraVideoDecoder::onDecodedFrame(gecko::camera::YCbCrFrame frame) {
-+  LOG("onDecodedFrame %llu", frame.timestampUs);
++void GeckoCameraVideoDecoder::onDecodedYCbCrFrame(const gecko::camera::YCbCrFrame *frame) {
```

Comment:
Would be good to assert here if one happens to pass null frame as that clearly the assumption.